fc/fractional_cover.py

```diff
@@ -103,6 +109,17 @@ def compute_fractions(nbar, regression_coefficients):
     :param numpy.array nbar: Input array of [green, red, nir, swir1, swir2] * (x, y)
     :return (numpy.array, numpy_array): Output array of [green, dead, bare] * (x, y), and the unmix error array
     """
+    if isinstance(nbar, dask_array_type):
+        lazy_compute_fractions = partial(_compute_fractions, regression_coefficients=regression_coefficients)
+        new_chunks = ((4,),) + nbar.chunks[1:]
+        out = dask.array.map_blocks(lazy_compute_fractions, nbar.rechunk({0: -1}), chunks=new_chunks, dtype='int8')
```

Comment:
For someone not intimately familiar with dask, I'm not quite sure what the above two lines are doing. Maybe worth adding a comment.